packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/metadataOutlineProvider.test.ts

```diff
@@ -176,6 +176,192 @@ describe('load org browser tree outline', () => {
     getCmpsStub.restore();
   });
 
+  it('should load folder nodes when Custom Objects type node is selected', async () => {
+    const expected = [
+      {
+        label: 'Account',
+        fullName: 'Account',
+        type: NodeType.Folder
+      },
+      {
+        label: 'AccountCleanInfo',
+        fullName: 'AccountCleanInfo',
+        type: NodeType.Folder
+      }
+    ];
+    const getCmpsStub = stub(
```

Comment:
nit: can we pull this out into a beforeEach for all these related tests? 
It'd look something like
```
let getComponentsStub: Sinon.Stub;

beforeEach(() => {
  getComponentsStub = MetadataOutlineProvider.prototype, 'getComponents');
});

afterEach(() => {
  getComponentsStub.restore(); // If you were using mocha sandbox functionality you could make this easier 
});

it('imma test', () => {
  getComponentsStub.returns(expected.map(n => n.fullName));
  ....
});